SchoolProject/src/main/java/Main.java

```diff
@@ -1,19 +1,27 @@
+import helpers.Grade;
 import models.School;
+import models.Student;
+import models.Subject;
 import repositories.StudentRepository;
+import repositories.TeacherRepository;
 import services.AccessValidator;
 import services.ReadDataService;
 import services.WriteDataService;
 
+import java.util.List;
 import java.util.Scanner;
 
 public class Main {
     public static void main(String args[]) {
```

Comment:
Looks like this method is very very big, so maybe you can refactor using the command pattern or other.